src/focus-visible.js

```diff
@@ -143,7 +144,12 @@ function applyFocusVisiblePolyfill(scope) {
     }
 
     if (hadKeyboardEvent || focusTriggersKeyboardModality(e.target)) {
+      //previous item should be unfocused before setting new focus
+      if(focusedElement){
+        focusedElement.blur();
```

Comment:
isn't it going to trigger the `blur` event twice ? 
maybe you should also check if the element is disabled ?

Reply:
or maybe you should just call ` removeFocusVisibleClass(focusedElement)` ?

Reply:
Updated. Call 'removeFocusVisibleClass(focusedElement)' is implemented.